src/libs/glad/CMakeLists.txt

```diff
@@ -1,3 +1,7 @@
+if(APPLE)
+  add_compile_options("-Wno-cast-function-type-mismatch")
```

Comment:
Maybe we could consider submitting a GLAD patch upstream to get rid of the warning at the source level, but that would be more involved.

Reply:
Fair enough. Could be fixed at the generator level, that's what I'm saying, but I understand if you don't feel the in inclination 😄 (I did submit patches in the past, and I also submitted a Nim codegen backend)